### PR TITLE
FEATURE: SD-760 geo pushpin chart documentation

### PR DESCRIPTION
**Pull request by naibinh**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/243/ : SUCCESS in 1m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/97/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/265/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/113/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/266/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/114/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/267/ : SUCCESS in 1m 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/115/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/268/ : SUCCESS in 1m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/116/ : SUCCESS in 5s

File: docs/geo_pushpin_chart_component.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        config={config}
+    />
+</div>
+```
+
+## Properties
+
+| Name | Required? | Type | Description |
+| :--- | :--- | :--- | :--- |
+| projectId | false | string | The project ID |
```
Comment (ptquang86) [inline docs/geo_pushpin_chart_component.md]: `projectId` is required

File: docs/geo_pushpin_chart_component.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+| location | true | [Attribute](afm.md#attribute) | An attribute definition that determines longitude and latitude of the pins |
+| segmentBy | true | [Attribute](afm.md#attribute) | An attribute definition that categorizes the pins |
```
Comment (ptquang86) [inline docs/geo_pushpin_chart_component.md]: `segmentBy` is optional

File: docs/geo_pushpin_chart_component.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+| tooltipText | false | [Attribute](afm.md#attribute) | Additional item that shows friendly label for location attribute instead of longitude and latitude |
+
+For common chart configs such as color, separators or legend visibility and location, refer to [Chart config](chart_config.md) for details
+
+The following example shows the supported `geoConfig` structure with sample values.
+```javascript
+{
+    points: {
+        sizeMin: '0.5x', // '0.5x' | '0.75x' | 'normal' | '1.25x' | '1.5x'  
+        sizeMax: '1.5x', // '0.5x' | '0.75x' | 'normal' | '1.25x' | '1.5x'
```
Comment (ptquang86) [inline docs/geo_pushpin_chart_component.md]: `minSize` and `maxSize`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/269/ : SUCCESS in 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/117/ : SUCCESS in 4s

File: docs/geo_pushpin_chart_component.md (hunk 3 above)
Comment (naibinh) [reply docs/geo_pushpin_chart_component.md]: Fixed

File: docs/geo_pushpin_chart_component.md (hunk 2 above)
Comment (naibinh) [reply docs/geo_pushpin_chart_component.md]: Fixed

File: docs/geo_pushpin_chart_component.md (hunk 1 above)
Comment (naibinh) [reply docs/geo_pushpin_chart_component.md]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/271/ : SUCCESS in 1m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/119/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/274/ : SUCCESS in 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/122/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/295/ : SUCCESS in 1m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/138/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/296/ : SUCCESS in 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/139/ : SUCCESS in 4s

**Comment by ptquang86** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/297/ : SUCCESS in 2m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/140/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/298/ : SUCCESS in 1m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/84/ : SUCCESS in 23s

**Pull request merged**